eos-update-flatpak-repos

```diff
@@ -130,6 +130,42 @@ def _remove_remotes():
                                    '--system', '--force', name])
 
 
+# dict of remote -> URLs and collection ID to migrate. If a collection
+# ID is not needed, None should be used.
+REMOTES_TO_MIGRATE = {
+    "eos-runtimes": {
+        "old-url": "https://ostree.endlessm.com/ostree/eos",
+        "new-url": "https://ostree.endlessm.com/ostree/eos-runtimes",
+        "new-collection-id": "com.endlessm.Runtimes",
+    },
+}
```

Comment:
It would be kind of nice to also specify `old-collection-id` and include that in the match so you could migrate the URL and/or the collection ID, but I didn't need that right now.